.github/workflows/node.js.yml

```diff
@@ -0,0 +1,37 @@
+# This workflow will do a clean installation of node dependencies, cache/restore them, build the source code and run tests across different versions of node
+# For more information see: https://docs.github.com/en/actions/automating-builds-and-tests/building-and-testing-nodejs
+
+name: Node.js CI
+
+on:
+  push:
+    branches: [ "master" ]
+  pull_request:
+    branches: [ "master" ]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [18.x, 20.x, 22.x]
+
+    steps:
+    - uses: actions/checkout@v4
+    # install system dependencies needed by the 'canvas' package
+    - name: Install dependencies for canvas
+      run: |
+        sudo apt-get update
+        sudo apt-get install -y libcairo2-dev libpango1.0-dev 
+        libjpeg-dev libgif-dev librsvg2-dev
```

Comment:
I think this should stay as one line; the error hints that GitHub’s environment is trying to perform the `libjpeg-dev` line _as a command_, and that would be due to its being on another line